src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,277 @@
+        System.out.println("    COMMAND: done x");
+        System.out.println("6.  Exit");
+        System.out.println("    COMMAND: bye");
+        printDividerLine();
+    }
+
+    private static void bye() {
+        printDividerLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        printDividerLine();
+    }
+
+    private static void printDividerLine() {
+        for (int i = 0; i < 30; i++) {
+            System.out.print("-");
+        }
+        System.out.println();
+    }
+
+    private static void addTask(String taskType, String taskDescription) {
+
+        if (taskDescription == null) {
+            printDividerLine();
+            System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+            printDividerLine();
+            return;
+        }
+
+        switch (taskType) {
+        case TODO:
+            taskList.add((new ToDo(taskDescription)));
+            break;
+        case DEADLINE:
+            if (!taskDescription.contains("/by")) {
+                printDividerLine();
+                System.out.println("Missing /by time!");
+                printDividerLine();
+                return;
+            }
+            taskList.add((new Deadline(taskDescription)));
+            break;
+        case EVENT:
+            if (!taskDescription.contains("/at")) {
+                printDividerLine();
+                System.out.println("Missing /at time!");
+                printDividerLine();
+                return;
+            }
+            taskList.add((new Event(taskDescription)));
+            break;
+        }
+
+        printDividerLine();
+        System.out.println("I have added this task:");
+        System.out.println("[" + taskList.get(numberOfTasks).getType() + "][" + taskList.get(numberOfTasks).getStatusIcon() + "] " + taskList.get(numberOfTasks).getDescription());
+        numberOfTasks++;
+        System.out.println("You have " + numberOfTasks + " task(s) in the list.");
+        printDividerLine();
+
+        try {
+            writeToFile();
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+    }
+
+    private static void displayList() {
+        printDividerLine();
+        if (numberOfTasks == 0) {
+            System.out.println("to-do list is empty! add something");
+            return;
+        }
+        System.out.println("The current to-do list is as follows:");
+        int counter = 1;
+        for (Task item : taskList) {
+            System.out.println(counter + ". [" + item.getType() + "][" + item.getStatusIcon() + "] " + item.getDescription());
+            counter++;
+        }
+        printDividerLine();
+    }
+
+    private static void markTaskComplete(String taskDescription) {
+        int taskNumber;
+
+        try {
+            taskNumber = Integer.parseInt(taskDescription);
+            if (taskNumber > numberOfTasks || taskNumber <= 0) {
```

Comment:
For compound expressions like this, declaring separate variables for each boolean expression would improve code readability.